### PR TITLE
added discord icon to the sidebar

### PR DESCRIPTION
**Pull request by evnxprk**
I was unable to find a way to view this on local so please let me know if these changes are good. If you can, please me know how to run this application on local because I was unable to find out.

### CODE REVIEW

**Comment by devlopersabbir** [comment]
@evnxprk thanks for your interest to contribute on [findDevs](https://finddevs.vercel.app), I really appreciate it 🥹🥰

#### Please read the contributing rules from this URL
**[CONTRIBUTING.md](https://github.com/devlopersabbir/find-devs/blob/default/CONTRIBUTING.md)** files

File: src/components/shared/sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import Link from "next/link";
 import { Github, LucideLinkedin } from "lucide-react";
 import { ModeToggle } from "../../theme/Toggle";
 import ProfileAdd from "./ProfileAdd";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faDiscord } from "@fortawesome/free-brands-svg-icons";
```
Comment (devlopersabbir) [inline src/components/shared/sidebar/Sidebar.tsx:9]: Please Keep using **[lucide-react](https://lucide.dev/icons/)**. I don't think we need to use any other `icon` library for only **discord** icon.

**Review by devlopersabbir — CHANGES_REQUESTED** [review]
Since `discord` icon is missing on **lucide** then we can use any `SVG` 😅

I have already created a [issue on lucide](https://github.com/lucide-icons/lucide/issues/2042) so, I hope they will add very soon

File: src/components/shared/sidebar/Sidebar.tsx (hunk 1 above)
Comment (evnxprk) [reply src/components/shared/sidebar/Sidebar.tsx:9]: Okay, I will do these by tonight or early tomorrow morning if that's okay with you?

Comment (devlopersabbir) [reply src/components/shared/sidebar/Sidebar.tsx:9]: Thanks for your response,🤩
no problem whenever you are free you can do it 🙃

**Pull request merged**